src/genie/libs/parser/iosxr/show_ospfv3.py

```diff
@@ -422,7 +422,7 @@ class ShowOspfv3VrfAllInclusiveNeighborDetailSchema(MetaParser):
                                             Any(): {
                                                 "interface": {
                                                     Any(): {
-                                                        Optional("bfd_enable"): bool,
+                                                        Optional("bfd_enable"): str,
```

Comment:
When changing existing parser schemas, it's better to make sure it will still work with legacy code using `Or`
```python
Optional("bfd_enable"): Or(str, int),
```